### PR TITLE
Convert print statements to print functions, even when commented

### PR DESCRIPTION
**Pull request by jaraco**
Fixes [#3408](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3408).

### CODE REVIEW

**Comment by jaraco** [comment]
oh, forgot newsfragment. standby.

File: misc/simulators/hashbasedsig.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def calculate(K, K1, K2, q_max, L_hash, trees):
             lg_q = lg(q_cand)
             lg_pforge = [lg_px[x] + (lg_q*x - lg_K2)*q_cand for x in xrange(1, j)]
             if max(lg_pforge) < -L_hash + lg(j) and lg_px[j-1] + 1.0 < -L_hash:
-                #print "K = %d, K1 = %d, K2 = %d, L_hash = %d, lg_K2 = %.3f, q = %d, lg_pforge_1 = %.3f, lg_pforge_2 = %.3f, lg_pforge_3 = %.3f" \
+                #print("K = %d, K1 = %d, K2 = %d, L_hash = %d, lg_K2 = %.3f, q = %d, lg_pforge_1 = %.3f, lg_pforge_2 = %.3f, lg_pforge_3 = %.3f" \)
```
Comment (itamarst) [inline misc/simulators/hashbasedsig.py:149]: Whatever tool you used to do this get the backslash case wrong.

File: misc/simulators/ringsim.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def dump_usage(self, numfiles, avg_space_per_file):
         # used is actual per-server ciphertext
         usedpf = [1.0*u/numfiles for u in used]
         # usedpf is actual per-server-per-file ciphertext
-        #print "min/max usage: %s/%s" % (abbreviate_space(used[-1]),
+        #print("min/max usage: %s/%s" % (abbreviate_space(used[-1]),)
```
Comment (itamarst) [inline misc/simulators/ringsim.py:133]: This is also wrong.

**Review by itamarst — COMMENTED** [review]
Should fix those two cases, otherwise looks fine. A very perfunctory review from core maintainers should suffice.

**Comment by itamarst** [comment]
And I guess validate there aren't any other cases of those two failure modes.

File: misc/simulators/hashbasedsig.py (hunk 1 above)
Comment (jaraco) [reply misc/simulators/hashbasedsig.py:149]: Thanks. I did a textual replacement then visually inspected, but not rigorously.

Comment (jaraco) [reply misc/simulators/hashbasedsig.py]: Fixed in 5a40bf4 and validated there aren't other cases of a line terminating in a `\` following a commented print.

File: misc/simulators/ringsim.py (hunk 2 above)
Comment (jaraco) [reply misc/simulators/ringsim.py]: Fixed in 5a40bf4 and confirmed there aren't other cases of `,)` appearing at the end of a commented print line.

**Comment by jaraco** [comment]
> And I guess validate there aren't any other cases of those two failure modes.

Good call. I scanned through the patch again and found no cases of a multi-line print statement.

**Review by sajith — APPROVED** [review]
I can't claim to be a core maintainer (not very confidently anyway), but I am wearing the reviewer hat this time. :-)

Looks good to me.  Please go ahead and merge.

**Pull request merged**